### PR TITLE
Add french lang

### LINKED ISSUE #189: Adding ability to choose language
Initial question to user to be able to select languages

### PR DESCRIPTION
**Pull request by lucymk**
- allows user to choose what language they want to speak in 
closes #189 

- adds language to state and sets according to user input using updateLang() function

- edits strings dependant on language in state

### CODE REVIEW

File: client/src/components/App.js [hunk 1]
```diff
@@ -62,21 +62,23 @@ export default class App extends React.Component {
...
-        speech: 'Little window welcome',
-        uniqueId: this.props.uniqueId
+        speech: 'Little Window language selection',
+        uniqueId: this.props.uniqueId,
+        lang: this.state.lang
       });
       this.setState({
-        uniqueId: this.props.uniqueId
+        uniqueId: this.props.uniqueId,
+        lang: this.state.lang
```
Comment (matthewdking) [inline client/src/components/App.js]: does `setState` need to set lang here seeing as it is setting it with the lang already in the state?

File: client/src/components/App.js [hunk 2]
```diff
@@ -207,6 +229,30 @@ export default class App extends React.Component {
       body: JSON.stringify(data)
     });
 
+  typingMessageLang = lang => {
+    if (lang === 'en') {
```
Comment (matthewdking) [inline client/src/components/App.js]: is it worth changing to objects with key value pairs now? If not then these could be one-liners
```js
if (lang === 'en') return 'typing...';
else if (lang === 'fr) return 'en train de taper...';
```

Comment (matthewdking) [reply client/src/components/App.js]: same below :point_down:

File: src/userMessage.js [hunk 3]
```diff
@@ -21,18 +23,27 @@ const apiaiCall = (req, res, speech) => {
...
       retrigger: '',
       timedelay: '',
-      refresh: '',
+      refresh: ''
     };
     // save message to database
     saveMessage(data.speech, response.sessionId);
 
     const payload = messages[1] ? messages[1].payload : {};
+    console.log('payload', payload);
```
Comment (matthewdking) [inline src/userMessage.js]: teeny log

**Comment by lucymk** [comment]
@matthewdking I think the changes have been somewhat addressed - I've not made json objects for the language stuff, but I have abstracted them out into a separate file which I think will be sufficient for now.

**Pull request merged**